docs/apigateway.md

```diff
@@ -0,0 +1,110 @@
+# Exposing your Actions via API Gateway (Experimental)
+
+In OpenWhisk you can invoke your actions using the [REST API](./reference.md#rest-api) using an HTTP Request only via a __POST__ method.
+This requires the HTTP client to make the request using the OpenWhisk authorization API key which is a master
+key that in addition of allowing to invoke an action, also allows for deleting and creating more actions.
+
+This experimental feature will allow you to invoke an action with HTTP methods other than POST, and not requiring to use 
+your OpenWhisk authorization API key from your HTTP client making the request.
+
+Use the CLI to expose your OpenWhisk actions through the OpenWhisk API Gateway. 
+
+## Expose an OpenWhisk action
+
+Let's expose a simple action that is already pre-installed with OpenWhisk
+
+```
+$ wsk api-experimental create /hello /echo get /whisk.system/utils/echo
+```
+```
+ok: created api /echo GET for action /whisk.system/utils/echo
+https://21ef035.api-gw.mybluemix.net/hello/echo
+```
+A new URL is generated exposing the `echo` action via a __GET__ HTTP method.
+
+Let's give it a try by sending a HTTP request to the URL.
+```
+$ curl https://21ef035.api-gw.mybluemix.net/hello/echo?marco=polo
+```
+This will invoke the `echo` action, returning back a JSON string with the parameters sent.
+```
+{
+  "marco":"polo"
+}
+```
+
+You can pass parameters to the action via simple query parameters, or via request body.
+
+### Exposing multiple actions
+
+Let's say you want to expose a set of actions for a book club for your friends.
+You have a series of actions to implement your backend for the book club:
+
+| action | http method | description |
+| ----------- | ----------- | ------------ |
+| getBooks    | GET | get book details  |
+| postBooks   | POST | adds a book |
+| putBooks    | PUT | updates book details |
+| deleteBooks | DELETE | deletes a book |
+
+
+Lets create an API for the Book club `Book Club` on basePath `/club` to manage the `books` resources.
```

Comment:
suggested rewording
->  `Let's create an API for the [b]ook club, named `Book Club`, with `/club` as its HTTP URL base path and `books` as its resource.`